app/models/ticket.rb

```diff
@@ -48,6 +48,15 @@ def self.total_price(conference, user)
     result ? result : Money.new(0, 'USD')
   end
 
+  def tickets_sold
+    TicketPurchase.where(ticket_id: id, conference_id: conference_id).inject(0) { |a, e| a + e.quantity }
+  end
+
+  def tickets_turnover
+    price = Money.new(price_cents, price_currency)
```

Comment:
I assume that you don't need this line, just ``tickets_sold * price`` should be fine!